src/platform/testing/e2e/cypress/support/commands/webComponents.js

```diff
@@ -0,0 +1,194 @@
+const FORCE_OPTION = { force: true };
+
+Cypress.Commands.add('fillTextWebComponent', (field, value) => {
+  if (value) {
+    const element =
+      typeof field === 'string'
+        ? cy.get(`va-text-input[name="${field}"]`)
+        : cy.wrap(field);
+
+    element
+      .shadow()
+      .find('input')
+      .as('currentElement')
+      .type(value);
+
+    cy.get('@currentElement').then(el => {
+      if (el.val()) cy.get(el).should('have.value', value);
+    });
+  }
+});
+
+Cypress.Commands.add('fillTextAreaWebComponent', (field, value) => {
+  if (typeof value !== 'undefined') {
+    const element =
+      typeof field === 'string'
+        ? cy.get(`va-textarea[name="${field}"]`)
+        : cy.wrap(field);
+
+    element
+      .shadow()
+      .find('textarea')
+      .as('currentElement')
+      .type(value);
+
+    cy.get('@currentElement').then(el => {
+      if (el.val()) cy.get(el).should('have.value', value);
+    });
+  }
+});
+
+Cypress.Commands.add('selectRadioWebComponent', (field, value) => {
+  if (typeof value !== 'undefined') {
+    cy.get(`va-radio-option[name="${field}"][value="${value}"]`)
+      .as('currentElement')
+      .click();
+    cy.get('@currentElement').should('have.attr', 'checked');
+  }
+});
+
+Cypress.Commands.add('selectYesNoWebComponent', (field, value) => {
+  const selection = value ? 'Y' : 'N';
+  cy.selectRadioWebComponent(field, selection);
+});
+
+Cypress.Commands.add('selectDropdownWebComponent', (field, value) => {
```

Comment:
this was the name that was used in the shared pattern, but open to renaming these differently.
These become `cy.selectDropdownWebComponent(field, value)` for example for the consumer